manifests/emissary/emissary-crds.yaml.in

```diff
@@ -1875,6 +1875,8 @@ spec:
               flush_interval_byte_size:
                 type: integer
               flush_interval_time:
+                description: 'TODO(lukeshu): In v3alpha2, change all of the `{foo}s`/`SecondDuration`
```

Comment:
Do these end up in the end-user provided CRD's?  Do we really want to have TODO comments there?

If so can we capture these somewhere else separately? In general, we should do analysis and come up with a plan for road to V3 CRD's. 

Oh and one other comment...Are we certain we will go to a v3Alpha2? If not then maybe drop putting a specific version?

Reply:
These do show up in the end-user CRDs.

If not v3alpha2 then a v3beta{X}; I think cur_version.last_segment++ gets the intent across clearly.

Reply:
yeah, ideally we do not ship with descriptions having TODO's.

I was just trying to avoid committing ourselves, if we were to skip to beta. I will let you make the final call.